client/tx/factory.go

```diff
@@ -39,6 +42,15 @@ func NewFactoryFromCLI(input io.Reader) Factory {
 		panic(err)
 	}
 
+	signModeStr := viper.GetString(flags.FlagSignMode)
+	signMode := signing.SignMode_SIGN_MODE_UNSPECIFIED
```

Comment:
Can we set it to default mode istead?

Reply:
At this point we do not have a `SignModeHandler` so we can't actually do that. So the best thing is to leave it as unspecified and the default gets set later once the `SignModeHandler` is provided.